lib/block_view_types.go

```diff
@@ -1980,7 +2116,7 @@ type MessagingGroupEntry struct {
 	// MessagingGroupMembers is a list of recipients in a group chat. Messaging keys can have
 	// multiple recipients, where the encrypted private key of the messaging public key
 	// is given to all group members.
-	MessagingGroupMembers []*MessagingGroupMember
+	MessagingGroupMembers []*MessagingGroupMember // Deprecated
```

Comment:
Should we rename to DeprecatedMessagingGroupMembers? Or something like that

Reply:
also if we drop access groups from message computation, we might just be able to delete this.